src/__tests__/shared/components/Sidebar.test.tsx

```diff
@@ -437,123 +438,81 @@ describe('Sidebar', () => {
     })
   })
 
-  /*
   describe('imagings links', () => {
     it('should render the main imagings link', () => {
-      const wrapper = setup('/imaging')
+      render('/imaging')
 
-      const listItems = wrapper.find(ListItem)
-      const imagingsIndex = getIndex(listItems, 'imagings.label')
-
-      expect(imagingsIndex).not.toBe(-1)
+      expect(screen.getByText(/imagings.label/i)).toBeInTheDocument()
     })
 
     it('should render the new imaging request link', () => {
-      const wrapper = setup('/imagings')
+      render('/imagings')
 
-      const listItems = wrapper.find(ListItem)
-      const imagingsIndex = getIndex(listItems, 'imagings.requests.new')
-
-      expect(imagingsIndex).not.toBe(-1)
+      expect(screen.getByText(/imagings.requests.new/i)).toBeInTheDocument()
     })
 
     it('should not render the new imaging request link when user does not have the request imaging privileges', () => {
-      const wrapper = setupNoPermissions('/imagings')
-
-      const listItems = wrapper.find(ListItem)
-      const imagingsIndex = getIndex(listItems, 'imagings.requests.new')
+      renderNoPermissions('/imagings')
 
-      expect(imagingsIndex).toBe(-1)
+      expect(screen.queryByText(/imagings.requests.new/i)).not.toBeInTheDocument()
     })
 
     it('should render the imagings list link', () => {
-      const wrapper = setup('/imagings')
-
-      const listItems = wrapper.find(ListItem)
-      const imagingsIndex = getIndex(listItems, 'imagings.requests.label')
+      render('/imagings')
 
-      expect(imagingsIndex).not.toBe(-1)
+      expect(screen.getByText(/imagings.requests.label/i)).toBeInTheDocument()
     })
 
     it('should not render the imagings list link when user does not have the view imagings privileges', () => {
-      const wrapper = setupNoPermissions('/imagings')
-
-      const listItems = wrapper.find(ListItem)
-      const imagingsIndex = getIndex(listItems, 'imagings.requests.label')
+      renderNoPermissions('/imagings')
 
-      expect(imagingsIndex).toBe(-1)
+      expect(screen.queryByText(/imagings.requests.label/i)).not.toBeInTheDocument()
     })
 
     it('main imagings link should be active when the current path is /imagings', () => {
-      const wrapper = setup('/imagings')
-
-      const listItems = wrapper.find(ListItem)
-      const imagingsIndex = getIndex(listItems, 'imagings.label')
+      const { container } = render('/imagings')
 
-      expect(listItems.at(imagingsIndex).prop('active')).toBeTruthy()
+      expect(container.querySelector('.active')).toHaveTextContent(/imagings.label/i)
```

Comment:
Just curious, any alternative to querySelector?

Reply:
We are currently trying to rewrite the tests like they were, therefore we are also testing implementation details, like checking the active element to have certain text inside of it.

This will eventually be changed later when we have converted all the tests, then we gonna improve the tests :tada: